vcd/resource_vcd_vapp_access_control.go

```diff
@@ -0,0 +1,360 @@
+package vcd
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+func resourceVcdAccessControlVapp() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAccessControlVappCreate,
+		Read:   resourceAccessControlVappRead,
+		Update: resourceAccessControlVappUpdate,
+		Delete: resourceAccessControlVappDelete,
+		Importer: &schema.ResourceImporter{
+			State: accessControlVappImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Required: false,
```

Comment:
not needed field, only Required or optional needed.

Reply:
Removed. It was a copy-and-paste operation, so I removed it also from all the resources that were using the same snippet